lib/gui/os/window-progress.js

```diff
@@ -59,4 +97,5 @@ exports.clear = () => {
   const ELECTRON_PROGRESS_BAR_RESET_VALUE = -1
 
   exports.currentWindow.setProgressBar(ELECTRON_PROGRESS_BAR_RESET_VALUE)
+  exports.currentWindow.setTitle(initialTitle)
```

Comment:
TBH, I'm not sure if I like the "asymmetry" of `.setTitle` here taking `initialTitle` as a parameter, but then the other `.setTitle` having `makeTitle(state)` as a parameter, with `makeTitle` itself retrieving `initialTitle`.
Instead of adding `initialTitle` back as a parameter `makeTitle`, I guess another approach would be to change _this_ line to `.setTitle(makeTitle(null))` (or `.setTitle(makeTitle({}))`), with `makeTitle` also modified as necessary, and that way the usage of `initialTitle` is entirely self-contained inside `makeTitle`? IYSWIM ;-)

Reply:
I agree with @lurch here, it'd be a little less confusing to just have `makeTitle(null)` returning the `initialTitle`, and would get rid of duplication of similar logic.
Also, what do you think of changing the name of `makeTitle()` to `getTitle()` or `getWindowTitle()` to clarify a bit what it's doing?

Reply:
@Shou do you want to change this ^ still?